app/components/service/includes/networks.html

```diff
@@ -8,12 +8,14 @@
       <table class="table" >
         <thead>
           <tr>
+            <th>Name</th>
             <th>ID</th>
             <th>IP address</th>
           </tr>
         </thead>
         <tbody>
           <tr ng-repeat="network in service.VirtualIPs">
+            <td>{{ network.Name }}</td>
             <td>
               <a ui-sref="network({id: network.NetworkID})">{{ network.NetworkID }}</a>
```

Comment:
I'd rather remove the ID column which is useless info actually. Instead, use a link on the name with the ID:

```
<a ui-sref="network({id: network.NetworkID})">{{ network.Name }}</a>
```